main.html

```diff
@@ -0,0 +1,333 @@
+                    style="width: 900px; height: 366px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <h1 id="Image" style="color: #CC3300;">Images</h1>
+                <div class="images">
+                    <img class="small-img" src="https://images.unsplash.com/photo-1603573355706-3f15d98cf100?ixid=MXwxMjA3fDB8MHxzZWFyY2h8OXx8cHJvamVjdHN8ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60"/>
+                    <img class="medium-img" src="https://images.unsplash.com/photo-1603573355706-3f15d98cf100?ixid=MXwxMjA3fDB8MHxzZWFyY2h8OXx8cHJvamVjdHN8ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60"/>
+                    <img class="large-img" src="https://images.unsplash.com/photo-1603573355706-3f15d98cf100?ixid=MXwxMjA3fDB8MHxzZWFyY2h8OXx8cHJvamVjdHN8ZW58MHx8MHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60"/>
+                </div>
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28155%2C155%2C155%2C1%29&t=cobalt&wt=sharp&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522small-img%2522%2520src%253D%2522%2522%252F%253E%250A%253Cimg%2520class%253D%2522medium-img%2522%2520src%253D%2522%2522%252F%253E%250A%253Cimg%2520class%253D%2522large-img%2522%2520src%253D%2522%2522%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+                    style="width: 420px; height: 258px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <h1 id="Typography" style="color: #CC3300;">Typography</h1>
+            <div class="typography">
+                <h1>This is biggest heading</h1>
+                <h2>This is 2nd biggest heading</h2>
+                <p>this is normal text it is free flowing as well</p>
+                <small>This is relatively small text</small>
```

Comment:
use div's with class names for this, instead of h1 tags. You should provide your custom size.